tests/saitests/py3/sai_qos_tests.py

```diff
@@ -456,7 +457,11 @@ def runTest(self):
             # queue 3/4  1                 1               1                                                1                                         1                         # noqa E501
             # queue 5    1                 1               1                                                1                                         1                         # noqa E501
             # queue 7    0                 1               1                                                1                                         1                         # noqa E501
-            assert (queue_results[QUEUE_0] == 1 + queue_results_base[QUEUE_0])
+            if platform_asic and platform_asic == "broadcom-dnx":
```

Comment:
Shall we revert them as fix from BRCM should fix this ?

Reply:
this commit has been reverted  #43740a5c0. The BRCM fix is provided. Testing is in progress for the fix